src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -4083,6 +4083,41 @@ static unsafe void Main(string[] args)
                 Diagnostic(RudeEditKind.UpdateAroundActiveStatement, "foreach (var a in G(a => a))", CSharpFeaturesResources.foreach_statement));
         }
 
+        [Fact]
+        public void ForEach_Update_Nullable()
+        {
+            string src1 = @"
+class C
+{
+    static void F()
+    {
+        var arr = new int?[] { 0 };
```

Comment:
>var arr = new int?[] { 0 }; [](start = 8, length = 27)

This is good. I'd also add a test where nullability changes by compiler inferring that it's no longer possible for the variable to be null (adding if check that returns).

Reply:
Will do. Thank you!